public/app/plugins/datasource/cloudwatch/language_provider.ts

```diff
@@ -37,26 +29,7 @@ type TypeaheadContext = {
   logGroupNames?: string[];
 };
 
-export function addHistoryMetadata(item: CompletionItem, history: CloudWatchHistoryItem[]): CompletionItem {
```

Comment:
Lots of code seemed to be just copied from loki and not used.

Reply:
Yeah, mentioned in the original CloudWatch PR that the language provider needed to be cleaned up so thanks for doing this.